tests/conftest.py

```diff
@@ -22,3 +23,8 @@ def sample_textfile(tmp_path: Path) -> Iterator[Path]:
 def existing_textfile() -> Path:
     existing_textfile_path = Path("tests") / "text_sample.txt"
     return existing_textfile_path
+
+
+@pytest.fixture(scope="session")
```

Comment:
One nitpick, session factories is easier to find if they are at the top of the file

Reply:
It's interesting, thank you for finding!